packages/amplication-client/src/ServiceTemplate/ServiceTemplateServiceList/ServiceTemplateServiceList.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  CircularProgress,
+  DataGrid,
+  EnumContentAlign,
+  EnumGapSize,
+  EnumItemsAlign,
+  EnumTextStyle,
+  FlexItem,
+  HorizontalRule,
+  Snackbar,
+  Text,
+} from "@amplication/ui/design-system";
+import { isEmpty } from "lodash";
+import { useEffect } from "react";
+import { EmptyState } from "../../Components/EmptyState";
+import { EnumImages } from "../../Components/SvgThemeImage";
+import { useAppContext } from "../../context/appContext";
+import PageContent from "../../Layout/PageContent";
+import NewServiceFromTemplateDialogWithUrlTrigger from "../../ServiceTemplate/NewServiceFromTemplateDialogWithUrlTrigger";
+import { formatError } from "../../util/error";
+import { pluralize } from "../../util/pluralize";
+import useServiceTemplate from "../hooks/useServiceTemplate";
+import { RESOURCE_LIST_COLUMNS } from "./ServiceTemplateServiceListDataColumns";
+
+const CLASS_NAME = "resource-list";
+const PAGE_TITLE = "Services";
+
+function ServiceTemplateServiceList() {
+  const { currentProject, currentResource } = useAppContext();
+
+  const {
+    findResourcesByTemplate,
+    findResourcesByTemplateLoading,
+    findResourcesByTemplateError,
+    findResourcesByTemplateData,
+  } = useServiceTemplate(currentProject);
+
+  useEffect(() => {
+    if (
+      !currentResource?.id ||
+      findResourcesByTemplateLoading ||
+      findResourcesByTemplateData
```

Comment:
Didn't you mean `!findResourcesByTemplateData` ? (with a `!` )

Reply:
nope, the idea is not to reload the data if the data already exist